nautobot/dcim/templates/dcim/controller_wireless.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'dcim/controller/base.html' %}
+{% load helpers %}
+
+{% block content %}
+<form method="post">
+    {% csrf_token %}
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>Wireless Networks</strong>
+            <div class="pull-right noprint">
+                {% if request.user.is_authenticated %}
+                    {% table_config_button_small wireless_networks_table %}
```

Comment:
Does table configuration work on this table in this view?

Reply:
Yes it does.